cli/src/commands/walletconnect.js

```diff
@@ -0,0 +1,286 @@
+// @flow
+/* eslint-disable no-console */
+/* eslint-disable no-fallthrough */
+import { scan, scanCommonOpts } from "../scan";
+import type { ScanCommonOpts } from "../scan";
+import { from } from "rxjs";
+import { first, tap, map, take } from "rxjs/operators";
+import { Observable } from "rxjs";
+import { log, listen } from "@ledgerhq/logs";
+import WalletConnect from "@walletconnect/client";
+import { getAccountBridge } from "@ledgerhq/live-common/lib/bridge";
+import { withDevice } from "@ledgerhq/live-common/lib/hw/deviceAccess";
+import signMessage from "@ledgerhq/live-common/lib/hw/signMessage";
+import { convertHexToNumber } from "@walletconnect/utils";
+import { BigNumber } from "bignumber.js";
+import { getCryptoCurrencyById } from "@ledgerhq/live-common/lib/currencies";
+
+type Opts = ScanCommonOpts &
+  $Shape<{
+    walletConnectURI: string,
+    walletConnectSession: string,
+    verbose: boolean,
+    silent: boolean,
+  }>;
+
+export default {
+  description: "Create a walletconnect session",
+  args: [
+    ...scanCommonOpts,
+    {
+      name: "walletConnectURI",
+      type: String,
+      desc: "WallecConnect URI to use.",
+    },
+    {
+      name: "walletConnectSession",
+      type: String,
+      desc: "WallecConnect Session to use.",
+    },
+    {
+      name: "verbose",
+      alias: "v",
+      type: Boolean,
+      desc: "verbose mode",
+    },
+    {
+      name: "silent",
+      type: Boolean,
+      desc: "do not output the proxy logs",
+    },
+  ],
+  job: (opts: Opts) =>
+    Observable.create(async (o) => {
```

Comment:
minor note: Observable.create can't take an async function. the unsub returned value will not be used.
you could eventually put the logic out in an async function outside, which is going to be the case when things get stabilized back into live-common